source/Calamari.Common/Plumbing/FileSystem/GlobExpressions/Ungrouper.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Calamari.Common.Plumbing.FileSystem.GlobExpressions
+{
+    public static class Ungrouper
```

Comment:
The name of this class is bothering me. I don't have a great suggestion, but it feels a bit ambiguous/generic

Reply:
I agree. What about `GlobExpressionGroupResolver`?

Reply:
Yeah, that's a better name 👍